h2o-r/h2o-DESCRIPTION.template

```diff
@@ -26,6 +26,7 @@ Imports: graphics,
 Suggests: ggplot2,
           mlbench,
           Matrix,
+          slam,
```

Comment:
Default as.h2o will fail for sparse data because the problem will be too big.

I think we need to be careful here, at least generate a warning with instructions what the user should do to speed things up.